src/components/App/App.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from 'react';
 import Search from '../Search/Search';
-import { getPokedexData } from '../../apiCalls'
-
+import { getPokedexData } from '../../apiCalls';
+import PokedexGrid from '../PokedexGrid/PokedexGrid';
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
       pokeDex: [],
-      foundPokemon: {}
+      foundPokemon: [],
```

Comment:
What does this data look like after the search is completed? Is it an array or some other type of data type. If it's a different data type than an array, we should initialize it from the start as the different data type

Reply:
Hi, I added as an array because we can use the `.length` to be able to render in the App page what it needs to be rendered ( for ex: if it is a error message or the found Pokemon)